### PR TITLE
Add Django GKE Polls Sample

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,5 @@ python-docs-samples.json
 __pycache__
 *db\.sqlite3
 managed_vms/django_tutorial/static/*
+container_engine/django_tutorial/static/*
```
Comment (jonparrott) [inline .gitignore]: Can you add this in the folder-level gitignore?

File: container_engine/django_tutorial/Makefile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+all: deploy
+
+.PHONY: create-cluster
+create-cluster:
+	gcloud container clusters create polls \
+		--scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform"
+
+.PHONY: create-bucket
+create-bucket:
+	gsutil mb gs://$(GCLOUD_PROJECT)
```
Comment (jonparrott) [inline container_engine/django_tutorial/Makefile]: Add the line to figure this out for them automatically: https://github.com/GoogleCloudPlatform/getting-started-python/pull/18/files#diff-854d0aab3e0cb4d4492bd8ecceecfb59R1

File: container_engine/django_tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
+# Getting started with Django on Google Container Engine
+
+This repository is an example of how to run a [Django](https://www.djangoproject.com/) 
+app on Google Managed VMs. It uses the [Writing your first Django app](https://docs.djangoproject.com/en/1
```
Comment (jonparrott) [inline container_engine/django_tutorial/README.md]: s/Managed VMs/Container Engine?

File: container_engine/django_tutorial/README.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+.9/intro/tutorial01/) Polls application as the example app to deploy. From here on out, we refe to this app as
```
Comment (jonparrott) [inline container_engine/django_tutorial/README.md]: Typo "refe".

File: container_engine/django_tutorial/README.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+Create a cluster for the bookshelf application:
+
+    gcloud container clusters create bookshelf \
+        --scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform" \
+        --num-nodes 2
+    gcloud container clusters get-credentials bookshelf
+
+The scopes specified in the `--scope` argument allows nodes in the cluster to access Google Cloud Platform APIs, such as the Cloud Datastore API.
+
+### Create a Cloud Storage bucket
```
Comment (jonparrott) [inline container_engine/django_tutorial/README.md]: Is this needed for the django app?

File: container_engine/django_tutorial/README.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+    docker build -t gcr.io/your-project-id/polls .
+    gcloud docker push gcr.io/your-project-id/polls
+
+Alternatively, this can be done using 
+
+    make build
+    make push
+    
+assuming you have the GCLOUD_PROJECT environment variable set.
```
Comment (jonparrott) [inline container_engine/django_tutorial/README.md]: This can be removed if you figure out the project for them.

File: .gitignore (hunk 1 above)
Comment (waprin) [inline .gitignore]: didn't realize you could do that, done, will do for MVM next.

File: container_engine/django_tutorial/Makefile (hunk 2 above)
Comment (waprin) [inline container_engine/django_tutorial/Makefile]: done

File: container_engine/django_tutorial/README.md (hunk 3 above)
Comment (waprin) [inline container_engine/django_tutorial/README.md]: done

File: container_engine/django_tutorial/README.md (hunk 4 above)
Comment (waprin) [inline container_engine/django_tutorial/README.md]: done

File: container_engine/django_tutorial/README.md (hunk 5 above)
Comment (waprin) [inline container_engine/django_tutorial/README.md]: I actually changed it so now Django will serve the files during debug. But obviously it's still better to serve from GCS, and we need it for the images in this case anyway.

File: container_engine/django_tutorial/README.md (hunk 6 above)
Comment (waprin) [inline container_engine/django_tutorial/README.md]: done

File: container_engine/django_tutorial/README.md (hunk 5 above)
Comment (jonparrott) [inline container_engine/django_tutorial/README.md]: Gotcha.

**Comment by jonparrott** [comment]
LGTM.

**Pull request merged**